### DISCUSSION TITLE
Setting up my media folders

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [pdulvp/jellyfin-qnap#38] [General]
I am trying to follow Jeff Geeling's video [https://www.youtube.com/watch?v=4VkY1vTpCJY](url) on setting up Jellyfin. He says that after installing we should see a new media folder on our NAS that we can then share and drag and drop media files too. This build seems to be different in that I don't see a new media folder in File Station on the QNAP. I then tried to add media librarys but I am not sure how to refer to a folder that I create in File Station.

I think I am probably totally on the wrong path and so would appreciate some guidance :)

**Comment by theincomlive** [comment]
have you tried path: /share/CACHEDEV1-2-3-or-4_DATA/your-folder
